CTP/shell/src/com/navercorp/cubridqa/shell/main/Main.java

```diff
@@ -57,7 +61,13 @@ public static void exec(String configFilename) throws Exception {
 		System.out.println("Continue Mode: " + context.isContinueMode());
 		System.out.println("Test Build: " + cubridPackageUrl);
 
-		if (cubridPackageUrl != null && cubridPackageUrl.trim().length() > 0) {
+		if (!CommonUtils.isEmpty(cubridPackageUrl)) {
+			if(!CommonUtils.checKURLIsAvailable(cubridPackageUrl)){
+				System.out.println();
+				System.out.println("[ERROR] Please confirm your build url is available!");
```

Comment:
Refine message:
as is:
[ERROR] Please confirm your build url is available!
to be:
[ERROR] Please check value ' "+cubridPackageUrl+"' given in parameter cubrid_download_url.